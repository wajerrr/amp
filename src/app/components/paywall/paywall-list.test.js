import React from 'react';
import renderer from 'react-test-renderer';
import PaywallList from './paywall-list';

describe('PaywallList', () => {
  it('should match the snapshot', () => {
    const paywallList = renderer
      .create(
        <PaywallList
          data={{
            type: 'list',
            content: {
              list: [
                {
                  title: 'Title 1',
                  text: 'Item 1',
                },
                {
                  title: 'Title 2',
                  text: 'Item 2',
                },
              ],
            },
          }}
        />
      )
      .toJSON();
    expect(paywallList).toMatchSnapshot();
  });
});
