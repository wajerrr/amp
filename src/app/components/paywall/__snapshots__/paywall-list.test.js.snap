// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PaywallList should match the snapshot 1`] = `
<section
  className={undefined}
>
  <ul
    className={undefined}
  >
    <li
      className={undefined}
    >
      <b>
        Title 1
      </b>
       - 
      Item 1
    </li>
    <li
      className={undefined}
    >
      <b>
        Title 2
      </b>
       - 
      Item 2
    </li>
  </ul>
</section>
`;
